get-started/part4.md

```diff
@@ -67,35 +67,41 @@ machine a swarm manager. From then on, Docker will run the commands you execute
 on the swarm you're managing, rather than just on the current machine.
 
 {% capture local-instructions %}
-You now have two VMs created, named `myvm1` and `myvm2` (as `docker-machine ls`
-shows). The first one will act as the manager, which executes `docker` commands
+You now have two VMs created, named `myvm1` and `myvm2`:
+
+```shell
+docker-machine ls
+```
+
+The first one will act as the manager, which executes `docker` commands
```

Comment:
All of the nodes execute `docker` commands. You can only execute management commands from manager nodes.